identity/src/main/java/com/stripe/android/identity/navigation/DriverLicenseScanFragment.kt

```diff
@@ -37,6 +37,7 @@ internal class DriverLicenseScanFragment(
                     startScanning(DL_BACK)
                 }
                 DL_BACK -> {
+                    continueButton.toggleToLoading()
```

Comment:
this change is added [to be in sync](https://github.com/stripe/stripe-android/blob/1a98a1729e4b4b69e4a7b2da483a4b55e12ba79f/identity/src/main/java/com/stripe/android/identity/navigation/IDScanFragment.kt#L40) with IDScanFragment